bazel/revisions.bzl

```diff
@@ -12,6 +12,7 @@ EMSCRIPTEN_TAGS = {
     "3.1.33": struct(
         hash = "49b960bd03b3a9da478a08541ce6eafe792a58a8",
         sha_linux = "eab02b3f4b7c076974452ba602f908a36adf597afa15b16095b441f191ede1bb",
+        sha_linux_arm64 = "5e15af6affcf37c9ce6c304b4aeccb87a2758e1ef029dbc996f9d77d7444378e",
```

Comment:
https://github.com/emscripten-core/emsdk/blob/a2ccccba58918706f815d806460d09310fe236d9/emscripten-releases-tags.json#L15
```
$ curl -sSL https://storage.googleapis.com/webassembly/emscripten-releases-builds/linux/49b960bd03b3a9da478a08541ce6eafe792a58a8/wasm-binaries-arm64.tbz2 | shasum -a 256
5e15af6affcf37c9ce6c304b4aeccb87a2758e1ef029dbc996f9d77d7444378e  -
```
This is the only newer release which seems to have linux arm64 binaries released.